integration-tests/OWNERS

```diff
@@ -0,0 +1,5 @@
+reviewers:
+  - jkopriva
```

Comment:
Shouldn't you also add approvers as reviewers?

Reply:
AFAIK approvers should also be able to review as per https://github.com/kubernetes/community/blob/master/community-membership.md#approver